src/number_theory/primorial.lean

```diff
@@ -155,8 +160,14 @@ begin
       ... = 4 ^ (2 * m + 1) : by ring
       ... = 4 ^ (n + 2) : by rw ←twice_m, },
   { cases lt_trichotomy 1 (n + 1) with one_lt_n n_lt_one,
-    { rw <- extend_prime_product_odd (n + 2) (by linarith) (by { rw [add_mod (n + 1) 1 2, h], norm_num, }),
-      calc ∏ i in filter prime (range (n + 2)), i
+    { have r : 2 < n + 1, by
```

Comment:
Why did this proof become longer?

Reply:
`linarith` stopped being able to provide this term. I don't know why. (I'm very bad at knowing when `linarith` is going to work; it's unable to prove quite a lot of things I would expect of it.)

Reply:
When did it stop being able to do this?

Reply:
You were right to flag this. The reason is because I inadvertently made the restrictions on `primorial_succ` more onerous than they used to be; we can fold the difficulty of this proof into `primorial_succ`.